docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
There is a Makefile

Reply:
I mean in the initial `cta-lstchain` folder, that is the one targeted in the ci deployment, the same way as:
https://github.com/cta-observatory/ctapipe/blob/master/Makefile

Reply:
Ah, right. Now I see it. I've always used directly `make html` instead.